#request constants
string location
---
#response constants
int8 status